Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:2-onbuild
+
+WORKDIR .
+RUN apt-get update && apt-get install -y \ 
+    sqlite3 \
+    libsqlite3-dev
+RUN pip install --trusted-host pypi.python.org -r requirements-test.txt
+RUN pip install -e . && echo "x509 Installed"
```

Comment:
why do we need `&& echo "x509 Installed"` if it's repeated in the subsequent line?